designs/administrator/future-openshift/air-gapped/index.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Configuring Air Gapped Environments
+This unit of work is a part of our continuous effort to improve the upgrade experience. Updates can be configured to receive information from Red Hat or via a custom update service. This set of enhancements allows customers to edit the Cincinnati server location directly in the console UI. 
+
+## 01 | Adding the ability to edit the Cincinnati server location via console to Cluster Settings details
+![1](img/1.png)
```

Comment:
This is nit picky but I guess I might have expected the "default configuration" part of the value to be part of the link as well, since it's set in the dialog too. Maybe some format where both parts appear as part of the value like

**Upstream configuration**
[Default configuration (https://apl.openshift.com/etc)](null) [pencil]

Reply:
I made a few updates - let me know what you think!